server/bot.ts

```diff
@@ -296,7 +344,7 @@ export class Bot {
     private tryHandleFeedbackMessage(message: Message, ctx: Context) {
         const msg = message as Message.CommonMessage;
         try {
-            if ((msg.reply_to_message as Message.TextMessage)?.text === PROMPT_FEEDBACK) {
+            if ((msg.reply_to_message as Message.TextMessage)?.text === this.i18n.t('bot.button.feedback')) {
```

Comment:
this is getting messy, I was thinking about adding a map of contexts for users and using it for such cases:
```
if (this.contexts.get(ctx.chat.id)?.givingFeedback) {
   ...
}
```
@ostrojen  you have written bots already, what's your suggestion?